hedera-node/src/main/java/com/hedera/services/txns/consensus/TopicUpdateTransitionLogic.java

```diff
@@ -204,7 +204,7 @@ private boolean canApplyNewAutoRenewAccount(ConsensusUpdateTopicTransactionBody
 			return true;
 		}
 
-		final var result = ledger.lookUpAccountId(op.getAutoRenewAccount(), INVALID_AUTORENEW_ACCOUNT);
+		final var result = ledger.lookUpAccountId(op.getAutoRenewAccount());
```

Comment:
Need to give `INVALID_AUTORENEW_ACCOUNT` as response

Reply:
Again, that will be handled just below this on [line 222](https://github.com/hashgraph/hedera-services/blob/a5f8542c353e4c86031d636d8fdb11adeb6ede52/hedera-node/src/main/java/com/hedera/services/txns/consensus/TopicUpdateTransitionLogic.java#L222)